src/pages/registration/files_and_license_tab/AdministrativeAgreementFilesRow.tsx

```diff
@@ -0,0 +1,82 @@
+import { Box, Checkbox, IconButton, TableCell, TableRow, Tooltip, Typography } from '@mui/material';
+import CancelIcon from '@mui/icons-material/Cancel';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import prettyBytes from 'pretty-bytes';
+import { Field, FieldProps, useFormikContext } from 'formik';
+import { AssociatedFile, AssociatedFileType } from '../../../types/associatedArtifact.types';
+import { ConfirmDialog } from '../../../components/ConfirmDialog';
+import { SpecificFileFieldNames } from '../../../types/publicationFieldNames';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { TruncatableTypography } from '../../../components/TruncatableTypography';
+
+interface AdminstrativeAgreementFilesRowProps {
+  file: AssociatedFile;
+  removeFile: () => void;
+  toggleLicenseModal: () => void;
+  baseFieldName: string;
+}
+
+export const AdminstrativeAgreementFilesRow = ({
```

Comment:
Skal være litt framsynt nå 😅 
det kommer til å bli en endring her, da begrepet "Administrative agreement" skal fjernes relativt snart, og erstattes med "UnpublishableFile". Dette er allerede delvis gjort, så tror vi kan ta den renamingen med en gang faktisk.
Husk å endre filnavn også. Og Props :p 

```suggestion
export const UnpublishableFileRow = ({
```

Reply:
Skaal det endres  i table header også?

Reply:
nei, det får vi ta når det kommer dit. Men fint å ta renaming nå, ellers hender det at git sliter med å forstå at det bare er endring på filnavn, slik at den viser at alt er nytt i stedet for bare noen få linjer når vi skal endre det senere